.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
```

Comment:
I wonder if this is really needed? I can already see people bypassing all the issue templates and just jotting down a "XY doesn't work!!1!" without any further repro info 😄

Reply:
Thanks. I didn't want to make these changes